app/models/study.rb

```diff
@@ -289,6 +300,36 @@ def search_data
     )
   end
 
+  # this method returns the first facility that has coords
```

Comment:
why is first facility coords needed vs all? Is it sorted by distance so this is the min distance?